services/consul/Dockerfile

```diff
@@ -18,7 +18,7 @@ mv dist ui && \
 rm consul_0.5.2_web_ui.zip 
 
 # Expose standard ports, as well as ports used by consul
-EXPOSE 8000 8301 8301/udp 8302 8302/udp 8400 8500 53/udp
+EXPOSE 8300 8301 8301/udp 8302 8302/udp 8400 8500 53/udp
```

Comment:
@jeremiaswerner port 8000 is not used by consul but probably nobody noticed. likely a typo of 8300.